packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -40,6 +40,7 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
     this._classNames = getClassNames(styles!, {
       theme: theme!,
       className: className || (calloutProps && calloutProps.className),
+      beakHeight: calloutProps && calloutProps.beakWidth,
```

Comment:
Why are you using different naming? the property is derived from beakWidth, so why not name it beakWidth?

Reply:
In my mind it made sense because I was working mostly with vertical scenarios, but you're right, I'll change it to be named `beakWidth` instead of `beakHeight`.